ppb/events.py

```diff
@@ -239,19 +238,28 @@ class SceneContinued:
     From the middle of the event lifetime that begins with
     :class:`SceneStarted`.
     """
-    scene: BaseScene = None  #: The currently running scene.
+    scene: BaseScene = None  #: The scene that is resuming operation.
```

Comment:
This is setting of a vocabulary/needlessly formal engineering speak flag in my head?

Reply:
`The scene that is resuming` enough you think?

Reply:
Sure.